src/test/skript/tests/regressions/pull-5748-fix bugs in the random number expression.sk

```diff
@@ -0,0 +1,8 @@
+
+# Test not related to a made issue. Details at pull request.
+test "random integer test":
```

Comment:
Since it's a random number do you think the tests should be looped a few times, since it's conceivable they randomly generate an OK number while having the possibility to generate a wrong number?

Reply:
This is a 2.7.0 target, any changes would need to be created in a new pull request for master.